test/parallel/test-fs-write-file-sync.js

```diff
@@ -28,39 +28,39 @@ if (common.isWindows) {
 common.refreshTmpDir();
 
 // Test writeFileSync
-var file1 = path.join(common.tmpDir, 'testWriteFileSync.txt');
+const file1 = path.join(common.tmpDir, 'testWriteFileSync.txt');
 
 fs.writeFileSync(file1, '123', {mode: mode});
 
 content = fs.readFileSync(file1, {encoding: 'utf8'});
-assert.equal('123', content);
+assert.strictEqual('123', content);
```

Comment:
Nit: can you please swap the arguments here and throughout the rest of the file?